### PR TITLE
Add coverage status and build instructions

### PR DESCRIPTION
**Pull request by kelunik**

### CODE REVIEW

**Comment by LeanderK** [comment]
looks good!

File: README.md [hunk 1]
```diff
@@ -1,7 +1,29 @@
-# Object-Service
+# object-service
```
Comment (LeanderK) [inline README.md:1]: Why did you change the capital letters of the object-service , but didn't with installation and Requirements?

**Comment by LeanderK** [comment]
do we really want to add the coverage status...it's a bit embarrassing ;)

File: README.md (hunk 1 above)
Comment (kelunik) [inline README.md:1]: Just because it's the repo name, can change it back if you want to.

**Comment by kelunik** [comment]
Yes… if it's embarrassing, add more tests, not remove the badge. ;-)

File: README.md (hunk 1 above)
Comment (LeanderK) [inline README.md:1]: i would capitalise it, it looks better. Also it's capital in our draft-report.

**Comment by LeanderK** [comment]
or, add the badge when its not 11% ;)

**Comment by kelunik** [comment]
See it as friendly reminder that we need more tests.

**Comment by marcelhollerbach** [comment]
does coverage status include integration tests ?

**Comment by kelunik** [comment]
Integration tests are not yet run during `gradle check` IIRC.

**Comment by marcelhollerbach** [comment]
Okay, looks fine to me, time to merge!
